pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[tool.black]
+line_length = 180
```

Comment:
180?!

That seems a little high - I saw you mention in one of Erik's PRs that everyone has wide monitors nowadays, but even so, it's a real PITA having long lines if the editor you're using doesn't support line wrapping. For the record, I prefer having several narrow windows side by side in my IDE, so not everyone uses wide monitors to view super-wide documents.

Reply:
What should it be set at?

Reply:
We've been using 100 as far as I know. There was a discussion around this a long time ago and that's where things came out, and it seems reasonable to me. Plus it's a nice number if you happen to have 10 fingers

Reply:
Is there a way to differentiate between set up line wrapping in your editor, so it wraps it while you are working on it, but then a different wrapping when you precommit? I set it to 120